packages/next/src/server/next.ts

```diff
@@ -214,10 +253,6 @@ export class NextServer {
   private async getServer() {
     if (!this.serverPromise) {
       this.serverPromise = this[SYMBOL_LOAD_CONFIG]().then(async (conf) => {
-        if (this.standaloneMode) {
-          process.env.__NEXT_PRIVATE_STANDALONE_CONFIG = JSON.stringify(conf)
-        }
-
```

Comment:
AFAIU, previously `standaloneMode` was only ever true for `NextCustomServer`, and this env var is only read in `loadConfig` (`this[SYMBOL_LOAD_CONFIG]()`), *after* this executed, so this does nothing useful.